cmd/utils/flags.go

```diff
@@ -493,6 +493,11 @@ var (
 		Usage: "Sets a cap on gas that can be used in eth_call/estimateGas (0=infinite)",
 		Value: ethconfig.Defaults.RPCGasCap,
 	}
+	RPCGlobalCallTimeoutFlag = cli.DurationFlag{
+		Name:  "rpc.calltimeout",
```

Comment:
Triage discussion: perhaps better to name it `rpc.evmtimeout` to disambiguate it from generic rpc calls

Reply:
Done.